packages/zone.js/lib/testing/fake-async.ts

```diff
@@ -137,14 +137,85 @@ Zone.__load_patch('fakeasync', (global: any, Zone: ZoneType, api: _ZonePrivate)
     return _getFakeAsyncZoneSpec().flush(maxTurns);
   }
 
+  /**
+   * set fake system time, in millis.
+   *
+   * @experimental
+   */
+  function setFakeSystemTime(time: number): void {
+    const zoneSpec = _getFakeAsyncZoneSpec();
+    zoneSpec.setCurrentRealTime(time);
+  }
+
+  /**
+   * get fake system time, in millis.
```

Comment:
```suggestion
   * Get fake system time, in millis.
```

please capitalize all the other ones as well.

Reply:
Got it, thanks.